packages/ChangesReloaded-Core.package/TimeStamp.extension/class/fromAuthorStamp..st

```diff
@@ -0,0 +1,12 @@
+*ChangesReloaded-Core
+fromAuthorStamp: aString
+	"Converts a string of the form '[author initials] [date] [time]', e.g. 'mg 6/29/2020 18:26' into an actual TimeStamp."
+
+	| tokens |
+	tokens := aString findTokens: Character separators.
+	^ tokens size = 3
+		ifTrue: [| dateString timeString |
+			dateString := tokens at: 2.
+			timeString := tokens at: 3.
```

Comment:
```suggestion
			dateString := tokens second.
			timeString := tokens third.
```
maybe this would work as well?

Reply:
Yep, this actually does work. `tokens` is an `OrderedCollection` which inherits from `SequencableCollection` which has accessors for `first` up to `ninth`.